ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -487,11 +497,15 @@ class ServicesHomeScreen extends React.Component<Props, State> {
           title: I18n.t("services.title"),
           body: () => <Markdown>{I18n.t("services.servicesHelp")}</Markdown>
         }}
+        isSearchAvailable={this.props.userMetadata !== undefined}
```

Comment:
It sounds a little weirds that search isn't available if user's metadata are undefined (but I understand why)
I think it is strictly connected to this story https://www.pivotaltracker.com/story/show/168758350
If you are agree I would place a TODO with story ref to remember to change that condition

Reply:
I set it in this way because, currently on master, the user can't display any tab if the user metadata are not available (and so the user cannot access any service detail).

Otherwise, the search works fine even if the user metadata aren't available. If you prefer, I enable search even if user metadata aren't available and the dedicated screen is displayed . What do you prefer?

cc @Undermaken @matteodesanti

Reply:
@valentinacala Let's check what @matteodesanti thinks about it